src/store/modules/rate/index.js

```diff
@@ -49,7 +50,10 @@ const actions = {
           .get(`${url}/getHistory?timestamp=${timestamp}`)
           .then((res) => {
             const rates = res?.data?.result[0]?.tickers ?? null
-            commit('setHistoryRates', { name: timestamp, value: rates })
+            if (timestamp !== undefined) {
```

Comment:
Why are you doing a request if timestamp === undefined?
May be the first line of getHistoryRates — `if (!timestamp) return;`?